service_dependencies.go

```diff
@@ -0,0 +1,34 @@
+/*
+ * Datadog API for Go
+ *
+ * Please see the included LICENSE file for licensing information.
+ *
+ * Copyright 2013 by authors and contributors.
+ */
+
+package datadog
+
+import (
+	"encoding/json"
+	"strconv"
+)
+
+type ServiceDependencies struct {
```

Comment:
I don't think this belons in here?

Reply:
whoops, forgot to push. Yeah I deleted that, good catch